vignettes/examples.Rmd

```diff
@@ -16,4 +16,146 @@ knitr::opts_chunk$set(
 
 ```{r setup}
 library(tlang)
+library(dplyr)
 ```
+
+# Demography Table
+For this demography table we are going to use `demog_data` an example analysis results table found in the package, which is based on the CDISC pilot data. This dataset has two different row label columns, `rowlbl1` and `rowlbl2` because we are going for a table with group and row labels. There are also two order columns which will be used to set the row order of the output. There is a single column to define our columns (multiple column columns are used when there is column spanning). Finally there is a param column, a value column and a grouping column, `grp`, which we can use for more complex controls.
+
+```{r, echo = FALSE}
+head(demog_data)
+```
+
+The mock we are going to match looks like this:
+
+![Mock deemographics table](images/gt_readme.png)
+
+For this table we have three dose columns, a total column and a p-value column as well as a mix of categorical and continuous analysis. 
+
+The first thing we are going to do when building out the `tfrmt` is specify all our columns 
+```{r}
+tfrmt(
+  # specify columns in the data
+  group = vars(rowlbl1,grp),
+  label = rowlbl2,
+  column = column, 
+  param = param,
+  values = value,
+  sorting_cols = vars(ord1, ord2)) %>% 
+  print_to_gt(demog_data)
+```
+
+While this make a table it isn't a very nice table and defiantly doesn't match the mock. So lets start with formatting all the numbers. To do this we are going to build a `body_plan` to add to our `tfrmt`.  This will be a fairly quick explanation of `body_plan`s but if you like more information see `vignettes("Body Plan")` 
```

Comment:
```suggestion
While this produces a table, it isn't a very nice table and definitely doesn't match the mock. So lets start with formatting all the numbers. To do this, we are going to build a `body_plan` to add to our `tfrmt`.  This will be a fairly quick explanation of `body_plan`s but if you like more information see `vignettes("Body Plan")` 
```